templates/security-series/index.html

```diff
@@ -0,0 +1,449 @@
+        image(
+          url="https://assets.ubuntu.com/v1/bf810a68-AppArmor_logo.svg",
+          alt="",
+          height="90",
+          width="100",
+          hi_def=True,
+          attrs={"class": ""},
+          loading="auto",
+        ) | safe
+      }}      
+      </p>   
+      <p class="p-heading--4">Securing Linux machines with AppArmor</p>
+      <p>AppArmor provides a crucial layer of security around applications. By providing the capability to whitelist an application’s permissible actions, AppArmor enables administrators to apply the principle of least privilege to applications.</p>
+      <a class="u-sv3" href="https://www.brighttalk.com/webcast/6793/440491">Resister</a>
+    </div>
+    <div class="col-6">
+      <p>
+        {{
+        image(
+          url="https://assets.ubuntu.com/v1/0f818a13-security-control.svg",
+          alt="",
+          height="80",
+          width="70",
+          hi_def=True,
+          attrs={"class": ""},
+          loading="auto",
+        ) | safe
+      }}      
+      </p>
+      <p class="p-heading--4">Ubuntu Core: A cybersecurity analysis</p>
+      <p>We provide substantial documentation and content to share and support the Ubuntu Core architecture and approach, but we don’t want anyone to have to take our word for it when choosing Ubuntu: We brought in Rule4 for an independent, third-party review of Ubuntu Core’s security architecture controls.</p>
+      <a class="u-sv3" href="https://www.brighttalk.com/webcast/6793/402503">Resister</a>
+    </div>
+  </div>
+</section>
+
+
+<section class="p-strip--light is-deep" id="multi-cloud-security">
+  <div class="row u-equal-height">
+    <div class="col-9">
+      <p class="p-heading--5">Topic</p>
+      <h2>Multi-cloud infrastructure security</h2>
+      <p class="u-sv2">This webinar series is for those interested in securing the full stack, from private and public cloud infrastructure to applications. </p>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <p>
+        {{
+        image(
+          url="https://assets.ubuntu.com/v1/bf810a68-AppArmor_logo.svg",
+          alt="",
+          height="90",
+          width="100",
+          hi_def=True,
+          attrs={"class": ""},
+          loading="auto",
+        ) | safe
+      }}      
+      </p>   
+      <p class="p-heading--4">Securing open source across multi-cloud environments</p>
+      <p>Join this webinar to see how you can ensure performant open source in production environments and achieve maximum availability by reducing downtime and providing access to high and critical CVE fixes. </p>
+      <p>Furthermore, learn how Ubuntu helps organisations remain compliant with government and industry standards and regulations, including Common Criteria EAL2 with FIPS 140-2 Level 1 certified crypto modules.</p>
+      <a href="https://www.brighttalk.com/webcast/6793/440517">Resister</a>
+    </div>
+    <div class="col-6">
+      <p>
```

Comment:
some of the space around the illustrations is off

![image](https://user-images.githubusercontent.com/441217/94435740-7209d780-0193-11eb-89d5-73fdb67b9bd1.png)


some are in `p` tags and some have `u-sv1` can you check them?